src/ApiStack.ts

```diff
@@ -374,6 +373,14 @@ export class ApiStack extends Stack implements Configurable {
         memorySize: 1024,
       },
     });
+
+    if (this.configuration.useZakenFromAggregatorAPI) {
```

Comment:
Wil je hier niet == 'true' in opnemen om er zeker van te zijn dat de waarde wordt gecontroleerd ipv of dit bestaat

Reply:
het is een boolean, die óf geset is en dan dus bewust true of false, of niet bestaat, en dan dus falsey is. gaat in alle gevallen goed (niet geset = false, wel geset is gekozen waarde).